torchci/lib/drciUtils.ts

```diff
@@ -45,6 +45,10 @@ export const EXCLUDED_FROM_FLAKINESS = [
   "linux-docs",
   "ghstack-mergeability-check",
 ];
+// Mapping the job to the list of suppressed labels
+export const SUPPRESSED_JOB_BY_LABELS: { [job: string]: string[] } = {
```

Comment:
IMO it should be limited to just pytorch/pytorch

Reply:
Sounds good, I can add that check